ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -497,6 +498,31 @@ module.exports = function CesiumMap(
       })
     },
     panToExtent(coords) {},
+    panToShapesExtent() {
+      if (!shapes.length) {
+        wreqr.vent.trigger('search:zoomToHome')
```

Comment:
Does this mean that if a search is run without geometries it will pan to the home area?

Reply:
Yeah. I wasn't sure how else to access the map's zoomToHome, so I used this.